client/src/stores/logux-client.ts

```diff
@@ -0,0 +1,59 @@
+import { badge, badgeEn, confirm, CrossTabClient, log } from '@logux/client'
+import { badgeStyles } from '@logux/client/badge/styles'
+import { atom, onMount } from 'nanostores'
+import { Client } from '@logux/client/client'
+
+import { subprotocol } from '../../../api'
+import { authStore, logout } from './auth'
+
+export const clientStore = atom<Client>()
+
+const loguxServer = `${
+  window.location.protocol.endsWith('s') ? 'wss' : 'ws'
+}://${window.location.host}/logux`
+
+onMount(clientStore, () => {
+  const client = new CrossTabClient({
+    subprotocol,
+    userId: '',
+    server: loguxServer
+  })
+
+  badge(client, { messages: badgeEn, styles: badgeStyles })
+  log(client)
+  confirm(client)
+
+  let started = false
+
+  clientStore.set(client)
+
+  let authUnsubscribe = authStore.subscribe(({ id }, changedKey) => {
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+    if (!changedKey || changedKey === 'id') {
+      if (id) {
+        client.changeUser(id)
+
+        if (started) {
+          client.node.connection.connect()
+        } else {
+          client.start()
+          started = true
+        }
+      } else if (client.node.connected) {
+        client.node.connection.disconnect()
```

Comment:
It is still not clear for me how can we clear fully the local copy of log when re-authentication happens.

Reply:
You need to `destroy` client and create new instance